src/axl.h

```diff
@@ -78,16 +72,8 @@ int AXL_Finalize (void);
  *              resume transfers after a crash.
  *
  * Returns an AXL ID, or negative number on error.
- *
- * NOTE: AXL_Create() used to only take in type and name.  The state_file arg
- * was added later.  We do some macro mangling to make sure both 2-arg and
- * 3-arg versions of AXL_Create() work, but the 2-arg version is deprecated
- * and should not be used.  If you're loading from an existing state_file, then
- * type must be the same as the type in state_file.
  */
-#define AXL_Create(type, name, ...) \
-        __AXL_Create(type, name, GET_ARG0(__VA_ARGS__))
-int __AXL_Create (axl_xfer_t xtype, const char* name, const char* state_file);
+int AXL_Create (axl_xfer_t xtype, const char* name, const char* state_file);
```

Comment:
Does this mean anyone still using the 2-arg version of `AXL_Create()`, they will now have issues?

Reply:
Yes, this change will break anyone with an ``AXL_Create(type, name)``, so they'll need to update to ``AXL_Create(type, name, NULL)``.  We'll need to update veloc and wemul.

Reply:
That's what I wanted to check. Thought @fahimcsebuet might want to know this change was happening. 😄

Reply:
@fahimcsebuet , as @CamStan noted, whenever you update AXL, you'll need to tweak your AXL_Init and AXL_Create lines from:
```
AXL_Init(NULL);
AXL_Create(type, name);
```
to
```
AXL_Init();
AXL_Create(type, name, NULL);
```